src/common/snippets/src/pass/softmax_decomposition.cpp

```diff
@@ -53,9 +53,12 @@ SoftmaxDecomposition::SoftmaxDecomposition() {
         const auto multiply = std::make_shared<ov::op::v1::Multiply>(exp, power);
 
         OPENVINO_ASSERT(axis < rank, "Softmax has incorrect axis");
-        std::vector<size_t> subtensor(rank, 1);
-        for (size_t i = axis; i < rank; ++i)
-            subtensor[i] = utils::get_full_dim_value();
+        const auto& subtensor = [&]() {
+            std::vector<size_t> subtensor(rank, 1);
+            for (size_t i = axis; i < rank; ++i)
+                subtensor[i] = utils::get_full_dim_value();
+            return subtensor;
+        }();
 
         PortDescriptorUtils::set_port_descriptor(power->input(0), subtensor);
         PortDescriptorUtils::set_port_descriptor(power->output(0), subtensor);
```

Comment:
```suggestion
        PortDescriptorUtils::set_port_descriptor(power->output(0), std::move(subtensor));
```
Instead make `subtensor` const and copy twice.

Reply:
Done